sch_fq.c

```diff
@@ -321,6 +467,33 @@ static struct fq_flow *fq_classify(struct sk_buff *skb, struct fq_sched_data *q)
 				     f->socket_hash != sk->sk_hash)) {
 				f->credit = q->initial_quantum;
 				f->socket_hash = sk->sk_hash;
+				//printk("flow hash in rb tree value of each flow is  : %u \n ",f->socket_hash );
+				if((pFlowid[0] == -1) && (pFlowid[1] ==-1) )
+				{
+				pFlowid[0] = f->socket_hash;
+				printk("flow pflowid 0 hash in rb tree value of each flow is  : %u \n ",pFlowid[0] );
+				/*if(pFlowid[0] == 0)
+					{
+				resetFlowid(pFlowid);
+					}*/
+				}
+				
+				if((pFlowid[0] != -1 ) && (pFlowid[1] == -1) )
+				{
+				
+				int lVal = valuePresentinArray(f->socket_hash,pFlowid);
+				
+				if(pFlowid[0] != f->socket_hash)
+				pFlowid[1] = f->socket_hash;
```

Comment:
Use one style to name variables consistently  -- `pFlowid` versus `pFlowId` versus `p_flow_id` (prefer the last one because kernel uses something similar too).

Reply:
Use capitalization and underscores consistently. `pFlowid` is not camel case, nor underscore style, and it can cause bugs if you use one different variable names in different places. Such bugs will be extremely hard to hunt down.